docs/bs-react-testing-library/examples.md

```diff
@@ -1,13 +1,40 @@
 ---
-id: example-intro
-title: Example
+id: examples
+title: Examples
 ---
 
+You can find more bs-dom-testing-library examples at
+[wyze/bs-dom-testing-library/src/\_\_tests\_\_](https://github.com/wyze/bs-dom-testing-library/tree/master/src/__tests__).
+
+You can find more bs-react-testing-library examples at
+[wyze/bs-react-testing-library/src/\_\_tests\_\_](https://github.com/wyze/bs-react-testing-library/tree/master/src/__tests__).
+
+## React Testing Library
+
+```reason
+/* Component_test.re */
+
+open Jest;
+open Expect;
+open ReactTestingLibrary;
+
+test("Component renders", () =>
+  <div style=ReactDOMRe.Style.make(~color="rebeccapurple", ())>
+    <h1> {ReasonReact.string("Heading")} </h1>
+  </div>
+  |> render
+  |> expect
```

Comment:
```suggestion
  |> container
  |> expect
```

Someone submitted this change to me yesterday because my example was wrong: https://github.com/wyze/bs-react-testing-library/pull/2

Reply:
Ok, I'll merge this and then you can follow up!